base_contact/__init__.py

```diff
@@ -19,4 +20,7 @@
 #
 ##############################################################################
 
-from . import base_contact
+import logging
+
+
+logging.getLogger(__name__).warn("This module is DEPRECATED. See README.")
```

Comment:
Warnings from runbot come from here, but it's a normal deprecation warning. Nothing to fix.